### PR TITLE
now that express is later in the stack to accommodate asset builds, w…

### PR DESCRIPTION
**Pull request by boutell**
…e have to rejigger to make sure routes are added after middleware

### CODE REVIEW

**Comment by boutell** [comment]
@abea this had to be merged as a hotfix, but please do review.

File: modules/@apostrophecms/ui/ui/apos/mixins/AposPublishMixin.js [hunk 1]
```diff
@@ -60,7 +60,6 @@ export default {
         }
         return false;
       }
-      return false;
```
Comment (boutell) [inline modules/@apostrophecms/ui/ui/apos/mixins/AposPublishMixin.js:63]: unrelated eslint.

File: defaults.js [hunk 2]
```diff
@@ -4,11 +4,11 @@ module.exports = {
     '@apostrophecms/util': {},
     '@apostrophecms/task': {},
     '@apostrophecms/schema': {},
-    '@apostrophecms/attachment': {},
     '@apostrophecms/asset': {},
     '@apostrophecms/launder': {},
     '@apostrophecms/http': {},
     '@apostrophecms/i18n': {},
+    '@apostrophecms/attachment': {},
```
Comment (boutell) [inline defaults.js:11]: This change has been reversed here, it was my first try at resolving the problem:

https://github.com/apostrophecms/apostrophe/commit/552dd6cd142438b3056ea504b5a7d566b1d11c88

The event-based change below in this PR is the keeper.

File: modules/@apostrophecms/express/index.js [hunk 3]
```diff
@@ -178,7 +178,7 @@ module.exports = {
         }
       },
       'apostrophe:modulesReady': {
-        async addCsrfAndModuleMiddleware() {
+        async addCsrfAndModuleMiddlewareAndRoutes() {
```
Comment (abea) [inline modules/@apostrophecms/express/index.js:181]: Seems like this is an indication this should be separate handlers.

**Pull request merged**